android/build.gradle

```diff
@@ -13,7 +13,16 @@ buildscript {
     }
 }
 
+def REACT_NATIVE_VERSION = new File(['node', '--print',"JSON.parse(require('fs').readFileSync(require.resolve('react-native/package.json'), 'utf-8')).version"].execute(null, rootDir).text.trim())
```

Comment:
Same thing using Groovy native JsonSlurper:
```
def REACT_NATIVE_VERSION = new groovy.json.JsonSlurper().parse(new File("$rootDir/../node_modules/react-native/package.json")).version
```
No need to run an external problem (Node.js) here, plus the line becomes significantly shorter too.

Reply:
sure.